### PR TITLE
Misc tidying

### PR DESCRIPTION
**Pull request by alexpearce**
Flesh out the index page so it's a nicer welcome, and tweak a little content that I was reading through.

### CODE REVIEW

File: 08-stripping.md [hunk 1]
```diff
@@ -1,55 +0,0 @@
----
```
Comment (ibab) [inline 08-stripping.md:1]: What happened to this lesson?
Do we want to remove it?

Comment (alexpearce) [inline 08-stripping.md:1]: [We do](https://github.com/lhcb/second-analysis-steps/pull/19/commits/c3ebef0475d04d4191ad655818ed42587076823c), as it's a dupe of the [LHCb data flow lesson](https://lhcb.github.io/first-analysis-steps/07-dataflow.html) in first-ana (which also contains more information).

Comment (apuignav) [inline 08-stripping.md:1]: +1

**Comment by ibab** [comment]
Looks good!

**Pull request merged**